.flake8

```diff
@@ -1,71 +1,19 @@
 [flake8]
 ignore =
-    B007,
-    B950,
-    E101,
-    E111,
-    E114,
-    E116,
-    E117,
-    E121,
-    E122,
-    E123,
-    E124,
-    E125,
-    E126,
-    E127,
-    E128,
-    E131,
-    E201,
-    E202,
-    E203,
-    E211,
-    E221,
-    E222,
-    E223,
-    E224,
-    E225,
-    E226,
-    E228,
-    E231,
-    E241,
-    E242,
-    E251,
-    E261,
-    E262,
-    E265,
-    E266,
-    E271,
-    E272,
-    E273,
-    E274,
-    E301,
-    E302,
-    E303,
-    E305,
-    E306,
-    E401,
-    E402,
-    E501,
-    E502,
-    E701,
-    E702,
-    E703,
-    E741,
-    F403,
-    W191,
-    W291,
-    W292,
-    W293,
-    W391,
-    W503,
-    W504,
-    I001,
-    I005,
-    I004
+    "B017", # assertRaises(Exception) - should be more specific
+    "B018", # useless expression, not assigned to anything
+    "B023", # function doesn't bind loop variable - will have last iteration's value
+    "B904", # raise inside except without from
+    "E402", # module level import not at top of file
+    "E501", # line too long
+    "E741", # ambiguous variable name
+    "F403", # can't detect undefined names from * import
+    "F405", # can't detect undefined names from * import
+    "F722", # syntax error in forward type annotation
+    "W191", # indentation contains tabs
+    "RUF001", # string contains ambiguous unicode character
+    "UP032", # Use f-string instead of `format` call (translations)
+    "UP030",
 
-per-file-ignores =
```

Comment:
Imports in __init__.py file can contain module level exports, might be a good idea to keep this `per-file-ignores` block.

Reply:
discussed: 
Ignore list can be updated afterwards if required